testserver/test/acceptance/pages/login_page.py

```diff
@@ -0,0 +1,26 @@
+__author__ = 'Kashif'
```

Comment:
I'm not sure we typically use these __author__ tags in Open edX code base. Normally we put our names in the CONTRIBUTORS file.